README.MD

```diff
@@ -4,30 +4,50 @@
 
 Decompilation in progress of Animal Crossing (GAFE01)
 
-An N64 version of the game is being worked on [here](https://github.com/zeldaret/af)
+A decompilation of the original N64 version of the game is being worked on [here](https://github.com/zeldaret/af).
 ## Cloning
 
 Use `--recursive` when cloning to have ppcdis in the repository. 
 
-## Build
-
-**It's preferible to use [wibo](https://github.com/decompals/wibo), a lightweight
-Wine replacement to work with the compilers, but Wine is still usable**
-
-### Building
-
-- Dump a copy of the game and extract **main.dol** and **foresta.rel.szs**.
-- Decompress **foresta.rel.szs** with yaz0 found in *tools/*.
-- Place **main.dol** and **foresta.rel** in *dump/*.
-- Place CodeWarrior 1.3.2 in *tools/1.3.2/* and 1.2.5n in *tools/1.2.5n/*.
-- Download [CodeWarrior 1.3.2r](https://mega.nz/file/WuBFTCLT#TmB5R4-1mEFkk4G1Vjn9_cHXRD9wOIH9CtOLaVSWEas) and extract it to *tools/1.3.2r/*.
-- Set the `N64_SDK` environmental variables with the path of your libultra or equivalent headers.
-	- Headers should be at `$N64_SDK/ultra/usr/include`.
-    - You may need to modify `Gpopmtx`'s `param` member to be `unsigned int`.
-- Install DevkitPPC, Ninja and Python:
-- Install Python modules from requirements.txt (`pip install -r requirements.txt`)
-- Run configure.py
-- Run ninja
+## Building
+
+### Docker (recommended)
```

Comment:
I think putting recommened/not recommended is a bit awkward. Since it implies that most of the users use this and it's the first option for asking help. I think it's better to leave them without this info, but my thoughts only

Reply:
I was kind of on the fence on adding those in the first place, but you're absolutely right. Will revise.